### DISCUSSION TITLE
weird bug(?) where map will play infinitely fast

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#17732] [Q&A]
I'm having an issue very occasionally (a couple times a month), that happens after I've had lazer open for a bit / playing a few maps.
I'm not exactly sure how to describe it, and I haven't managed to remember to record a video, but pretty much I'll start a map and it will run extremely fast - going from me clicking the map name in the list to fail screen in ~a second - and the song will also be sped up the same amount.
also, when this happens, I'm not able to close the game; it hangs on a black screen until I manually kill the osu! process.
I know this isn't nearly enough information for a bug report, but I'm attaching logs either way:
[runtime.log](https://github.com/ppy/osu/files/8455608/runtime.log)
[performance.log](https://github.com/ppy/osu/files/8455609/performance.log)
[network.log](https://github.com/ppy/osu/files/8455610/network.log)
[database.log](https://github.com/ppy/osu/files/8455611/database.log)

**Comment by ghost** [comment]
I just managed to get a video of this:

https://user-images.githubusercontent.com/64013111/162594088-fd484b49-51c1-4a2a-a08d-70ae246172cc.mov

(my mic isn't broken, that's actually the sound that the game made - it didn't do that yesterday though, and not always when the audio breaks like this it also breaks map speed)

**Reply by peppy** [reply]
This is going to be something related to your linux drivers or system.

```
2022-04-08 21:37:28 [important]: Thread Audio failed to exit in allocated time (30000ms).
```

hints at this. nothing we can do sorry.
